### DISCUSSION TITLE
Accessing the console output using cflib

### DISCUSSION THREAD

**Discussion by steverpalmer** [bitcraze/discussions#1178] [Q&A] [answered]
I'm a newbie to crazyflie, so I apologise if this is a stupid question.

What is the simplest python program using the cflib library to print the console log to the screen?

I want to write my own python client for my crazyflie 2.1, and think that it would be useful to be able to report the crazyflie console output (after a crash or two).  I've looked at https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/crazyflie/console/, but it doesn't mean much to me - sorry.  Ideally, I'd like to see the simplest complete python program that will connect to the crazyflie and print the console output until it is terminated with a Ctrl-C (or something like).

P.S. I can't use the cfclient stuff to see the console at present because I'm on an ARM64 which doesn't like pyqt6.

Thanks.

**Comment by knmcguire** [comment, marked as answer]
Hi! Is this the script what you are looking for?

https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/console/console.py

**Reply by knmcguire** [reply]
Also you can try out the headless client (although we haven't used this for a while): https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/cfheadless/

**Reply by steverpalmer** [reply]
Just the thing. Thanks.
